src/constants.ts

```diff
@@ -85,6 +77,10 @@ export const CONSTANTS = {
             "error.invalidPythonPath",
             'We found that your selected Python interpreter version is too low to run the extension. Please upgrade to version 3.7+ or select a different interpreter (CTRL+SHIFT+P and type "python.selectInterpreter") and restart the application.'
         ),
+        LESS_PYTHON_VERSION_FOR_MICROBIT_DEPLOYMENT: localize(
```

Comment:
_&ast;nit&ast;_
`LOW` instead of `LESS`?

Reply:
will update it